lib/locales/urhobo/names/male_first_name.js

```diff
@@ -0,0 +1,53 @@
+module.exports = [
+  'Edewor',
+  'Eseoghene',
+  'Oghenebrume',
+  'Oghenevwogagan',
+  'Oghenochuko',
+  'Onakpoma',
+  'Onomavwe',
+  'Onomine',
+  'Efetobore',
+  'Ogbamremu',
+  'Oboganriemu',
+  'Ojakorotu',
+  'Onanojah',
+  'Udumebraye',
+  'Edafetanure',
+  'Efemena',
+  'Efemuaye',
+  'Efetobore',
+  'Etanomare',
+  'Etaredafe',
+  'Ighomuedafe',
+  'Ighovavwerhe',
+  'Okagbare',
+  'Odafe',
+  'Onogaganmue',
+  'Orodena',
+  'Akpofure',
+  'Edaferierhi',
+  'Edemirukaye',
+  'Emetitiri',
+  'Emudiaga ',
+  'Emuobosa',
+  'Enakpodia',
+  'Otevwobrise',
+  'Udumebraye',
+  'Okoro ',
+  'Okotete',
+  'Ohwofasa ',
+  'Igberaharha',
+  'Etaidaferua ',
+  'Ighovavwerhe',
+  'Uduefebraye',
+  'Oyibo',
+  'Oyibonanarhoro ',
+  'Oyiborhoro',
+  'Oyiborode',
+  'Uvo',
+  'Ogoro',
+  'Uloho',
+  'Okpara',
+];
+  
```

Comment:
You might want to lose the trailing spaces on this line, that's what's causing this red warning here.